packit_service/worker/jobs.py

```diff
@@ -284,11 +261,10 @@ def process_jobs(self, event: Event) -> List[TaskResults]:
             )
             # check whitelist approval for every job to be able to track down which jobs
             # failed because of missing whitelist approval
-            whitelist = Whitelist()
-            user_login = getattr(event, "user_login", None)
-            if user_login and user_login in self.service_config.admins:
-                logger.info(f"{user_login} is admin, you shall pass.")
-            elif not whitelist.check_and_report(
+            whitelist = (
+                Whitelist()
+            )  # TODO: Why do we create separate instance for each iteration?
```

Comment:
It's maybe because of some refactorings. I don't think we need to create multiple instances.

I am thinking if this can't be moved away (=before) the for loop? WDYT?

Reply:
I don't see anything shared, but there was/is a comment for that, so I'm just asking :D

Reply:
I am now not sure.

We want to have a result for each handler, but if there are multiple handlers, we will run validation multiple times and potentially add multiple comments. Am I right?

Reply:
I'll move it out of the for-loop
it doesn't matter really, cause first failure collects `processing_results` for each `job_config` and returns them